library/std/src/io/readbuf.rs

```diff
@@ -178,8 +178,9 @@ impl<'a> ReadBuf<'a> {
     ///
     /// Panics if the filled region of the buffer would become larger than the initialized region.
     #[inline]
-    pub fn add_filled(&mut self, n: usize) {
+    pub fn add_filled(&mut self, n: usize) -> &mut Self {
         self.set_filled(self.filled + n);
+        self
```

Comment:
```suggestion
        self.set_filled(self.filled + n)
```

This should work, right?

Reply:
It does! Thanks!